cfme/infrastructure/pxe.py

```diff
@@ -704,6 +687,39 @@ def delete(self, cancel=True):
             navigate_to(self, 'Details')
 
 
+@attr.s
+class SystemImageTypeCollection(BaseCollection):
+    """ Collection class for SystemImageType. """
+
+    ENTITY = SystemImageType
+
+    def create(self, name, provision_type, cancel=False):
+        """
+        Creates a System Image Type object
+
+        Args:
+            name: It contains name of the System Image Type
+            provision_type: Type on Image. i.e Vm and Instance or Host
+            cancel (boolean): Whether to cancel out of the creation.  The cancel is done
+                after all the information present in the SIT has been filled in the UI.
+        """
+
+        system_image_type = self.instantiate(name, provision_type)
+
+        view = navigate_to(self, 'Add')
+        view.fill({'name': name, 'type': provision_type})
+        if cancel:
+            view.cancel.click()
+            msg = 'Add of new System Image Type was cancelled by the user'
+        else:
+            view.add.click()
+            msg = 'System Image Type "{}" was added'.format(name)
+        main_view = self.create_view(PXESystemImageTypesView)
+        main_view.flash.assert_success_message(msg)
+        return system_image_type
+
+
+@navigator.register(SystemImageTypeCollection, 'All')
 @navigator.register(SystemImageType, 'All')
```

Comment:
**Required**

Remove the registration against SystemImageType, the entity object doesn't have an 'All' page

Reply:
@mshriver: It is required for details page navigation.

Reply:
How is it required?

Navigating to details can be done with a prerequisite of `NavigateToAttribute('parent', 'All'), since the entity object has a 'parent' attribute that is the collection object.